client/src/main/java/io/pravega/client/segment/impl/SegmentMetadataClientImpl.java

```diff
@@ -49,7 +49,7 @@
 @RequiredArgsConstructor
 @Slf4j
 class SegmentMetadataClientImpl implements SegmentMetadataClient {
-    private static final RetryWithBackoff RETRY_SCHEDULE = Retry.withExpBackoff(1, 10, 9, 30000);
+    private static final RetryWithBackoff RETRY_SCHEDULE = Retry.withExpBackoff(1, 10, 10, 30000);
```

Comment:
should we increase this to a larger retry count so that we reduce the chances of encountering the issue even on slower systems?